### DISCUSSION TITLE
Are there any examples of uploading PDFs and, say, storing that in a Cloud Storage bucket?

### DISCUSSION THREAD

**Discussion by thomasmost** [tokio-rs/axum#2422] [Q&A] [answered]
### Summary

Would that just be standard form/multipart? Or "application/pdf" somehow?

Looking for guidance

### axum version

0.6.20

**Comment by davidpdrsn** [comment, marked as answer]
Since this isn’t specifically about axum, I’d recommend you ask in a more general web dev forum. You’ll probably get better advice that way.

If you’re just uploading pdfs to save them in a bucket then I don’t suppose the content type matters but just the data itself.
